.circleci/real_config.yml

```diff
@@ -1987,11 +1987,13 @@ jobs:
       image: <<pipeline.parameters.machine-image>>
     resource_class: xlarge
     steps:
-      - run: echo 'export PW_SERVER_ADDRESS="http://localhost:3001"' >> $BASH_ENV # DNJ TODO -remove when tested
-      - run: echo 'export PW_USER_NAME=admin' >> $BASH_ENV
-      - run: echo 'export PW_PASSWORD=' >> $BASH_ENV
-      - run: echo 'export DET_WEBPACK_PROXY_URL="http://localhost:8082"' >> $BASH_ENV
-      - run: echo 'export DET_WEBSOCKET_PROXY_URL="ws://localhost:8082"' >> $BASH_ENV
+      # - run: echo 'export PW_DET_PATH="/tmp/venv/bin/det"' >> $BASH_ENV
```

Comment:
these have moved to playwright context. I thought it might be good to comment the values for now since circle ci hides them. is there any other place we should store them for reference?

Reply:
I would just delete them here. Add them to the readme if not there already?